gui/src/main/java/org/jboss/as/console/mbui/widgets/ModelNodeFormBuilder.java

```diff
@@ -649,17 +649,23 @@ public int compare(Property property, Property property1) {
                     // iterate over the "requires" attribute list
 
                     boolean sourceItemDefined = isFormItemDefined(sourceFormItem);
```

Comment:
As I was trying to explain this doesn't fix the problem that may occur with `requires`. Returning false for null item is not correct in that context.